cmd/ebrelayer/txs/signature.go

```diff
@@ -21,13 +22,15 @@ func LoadPrivateKey() (key *ecdsa.PrivateKey, err error) {
 	// Private key for validator's Ethereum address must be set as an environment variable
 	rawPrivateKey := os.Getenv("ETHEREUM_PRIVATE_KEY")
 	if strings.TrimSpace(rawPrivateKey) == "" {
-		log.Fatal("Error loading ETHEREUM_PRIVATE_KEY from .env file")
+		log.Println("Error loading ETHEREUM_PRIVATE_KEY from .env file")
+		return &ecdsa.PrivateKey{}, errors.New("can't load ethereum private key from .env file")
 	}
 
 	// Parse private key
 	privateKey, err := crypto.HexToECDSA(rawPrivateKey)
 	if err != nil {
-		log.Fatal(err)
+		log.Println(err)
+		return privateKey, err
```

Comment:
I feel like this line should be:
```
return nil, err
```
due to the function signiture using a pointer.

Reply:
Accepted.